src/lib/arch/aarch64/Instruction_execute.cc

```diff
@@ -1611,6 +1611,12 @@ void Instruction::execute() {
         results[0] = static_cast<int64_t>(std::trunc(operands[0].get<float>()));
         break;
       }
+      case Opcode::AArch64_FCVTZUv1i64: {  // fcvtzu dd, dn
+        // TODO: Handle NaNs, denorms, and saturation
```

Comment:
Is this something that would be difficult "TODO" now?

Reply:
I think this has been present for some time (TODO now included in 10 instructions). Having looked into possible handling, it is probably best to do this in a separate PR in order to account for rounding modes, handling of NaNs, inf, etc.